moped-editor/src/views/projects/projectsListView/useProjectListViewQuery/useCsvExport.js

```diff
@@ -29,6 +35,42 @@ export const CsvDownloadDialog = ({ dialogOpen }) => (
   </Dialog>
 );
 
+export const CsvDownloadOptionsDialog = ({
+  dialogOpen,
+  handleDialogClose,
+  handleContinueButtonClick,
+  handleRadioSelect,
+}) => (
+  <Dialog open={dialogOpen} onClose={handleDialogClose}>
+    <DialogContent>
+      <FormControl>
+        <RadioGroup defaultValue={"visible"} onChange={handleRadioSelect}>
```

Comment:
you've opted to go with an "uncontrolled" component in this case. by that i mean that this component does not use the `columnDownloadOption` as a prop, it simply uses a handler function to set the value of `columnDownloadOption` on change.

that works totally fine for these purposes, but, in general, i almost always go with controlled form components, because i've found that over time i often end up re-visiting uncontrolled components when i want to add more complex / reactive behavior. 

for example, if we wanted add a "reset" button this to dialog that resets the radio button to the "default" value, we would not be able to do that without turning this into a controlled component.

if you want to put this into action, it is as easy as providing the `columnDownloadOption` as a prop to `CsvDownloadOptionsDialog`, and then passing that prop to `value` prop of the `RadioGroup` component.

once you do that, one effect that you'll notice is that radio selection  is persisted when you close the modal and open again.

that's my long-winded way of saying that i would prefer to see you make this a controlled component, but it is not critical. there are some [React docs](https://react.dev/learn/sharing-state-between-components#controlled-and-uncontrolled-components) on this subject that illustrate this much better than i have.

as one last aside, one of the reasons [React Hook Form](https://react-hook-form.com/) is so popular is because writing controlled components for multiple form inputs quickly becomes absurdly cumbersome 😵